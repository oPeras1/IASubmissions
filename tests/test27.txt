1	1	1	1	1	2	2	2	2	2	3	3	3	4	4	5	5	6	6	6
7	7	7	7	1	2	8	8	8	2	3	4	4	4	4	5	9	9	10	6
11	7	7	12	2	2	8	8	2	13	13	4	4	9	9	5	9	10	10	6
11	12	12	12	2	2	2	2	2	13	13	14	9	9	9	9	9	10	6	6
11	11	11	11	11	11	15	15	13	13	14	14	16	9	9	9	6	6	6	17
11	18	11	11	11	15	15	15	19	14	14	14	16	16	16	16	6	17	17	17
18	18	11	18	20	15	15	19	19	19	14	21	21	16	16	22	22	17	23	23
24	18	18	18	20	25	25	25	26	26	27	27	21	16	22	22	22	17	17	23
24	24	24	20	20	25	25	24	24	26	27	27	21	22	22	22	27	17	17	23
24	24	24	24	24	24	24	24	26	26	26	27	27	27	27	27	27	28	17	17
29	30	30	24	31	24	32	24	26	33	34	34	34	27	35	27	28	28	17	17
29	30	30	31	31	31	32	32	32	33	34	34	36	37	35	28	28	35	35	35
29	29	30	38	38	32	32	33	33	33	33	36	36	37	35	35	35	35	39	40
41	29	38	38	38	38	42	42	42	33	36	36	37	37	35	35	39	39	39	40
41	41	41	38	38	43	43	42	42	33	37	37	37	37	39	39	39	44	40	40
41	45	41	41	41	43	42	42	46	37	37	47	47	47	48	49	44	44	44	44
45	45	41	41	41	43	43	43	46	46	47	47	50	48	48	49	44	44	51	44
45	52	52	52	52	43	53	53	53	46	46	47	50	48	49	49	51	51	51	44
54	52	54	52	55	53	53	53	53	53	53	50	50	48	51	49	51	51	56	44
54	54	54	54	55	55	55	55	53	53	50	50	48	48	51	51	51	56	56	56