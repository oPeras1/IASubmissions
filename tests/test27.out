1	1	L	L	L	2	2	2	2	2	L	L	L	4	4	L	L	6	6	6
T	T	T	7	L	2	T	T	T	2	L	4	S	S	4	L	9	9	S	6
11	T	7	L	2	L	8	T	2	L	13	S	S	9	S	L	9	S	S	6
11	L	L	L	2	L	L	L	2	L	13	T	9	S	S	9	9	S	6	6
11	11	11	S	S	11	S	15	L	L	T	T	16	S	9	9	L	L	L	17
11	T	S	S	11	S	S	15	T	14	14	T	16	T	T	T	L	17	17	17
T	T	11	18	L	S	15	T	T	T	14	L	L	16	T	22	22	17	L	L
24	T	18	18	L	25	S	S	26	L	27	27	L	16	S	S	22	S	17	L
24	S	S	L	L	S	S	24	24	L	27	27	L	S	S	22	L	S	S	L
S	S	24	24	24	24	24	24	L	L	26	27	27	27	L	L	L	28	S	17
29	L	L	24	T	24	L	24	26	I	T	T	T	27	I	27	S	S	17	17
S	30	L	T	T	T	L	L	L	I	34	T	36	37	I	S	S	35	35	35
S	S	L	38	38	32	32	33	33	I	33	S	S	37	I	35	35	35	39	L
41	S	38	T	T	T	L	L	L	I	S	S	37	L	I	35	S	S	39	L
41	41	L	38	T	43	43	42	L	33	37	L	L	L	39	S	S	44	L	L
41	S	L	L	L	43	42	42	S	37	37	T	47	47	48	L	44	S	S	44
S	S	41	41	41	L	L	L	S	S	T	T	50	I	48	L	S	S	51	44
S	52	T	T	T	L	53	53	53	S	46	T	50	I	L	L	51	51	51	44
L	52	54	T	55	53	53	L	L	L	53	S	S	I	51	49	L	51	T	44
L	L	L	54	I	I	I	I	53	L	S	S	48	I	L	L	L	T	T	T